src/v3/src/components/Widget/style.module.css

```diff
@@ -0,0 +1,159 @@
+/*! CSS Used from: Embedded */
+:global(.bgStyle) {
+  background-image: none;
+}
+/*! CSS Used from: https://op3static.oktacdn.com/assets/js/sdk/okta-signin-widget/7.3.1/css/okta-sign-in.min.css */
+:global(.clearfix) {
+  display: block;
+}
+:global(.clearfix:after,
+.clearfix:before) {
+  display: block;
+  block-size: 0;
+  clear: both;
+  content: ".";
+  visibility: hidden;
+}
+:global(.hide) {
+  display: none;
+}
+:global(.auth .content) {
+  min-block-size: 100%;
+  min-inline-size: 300px;
+  display: inline-block;
+  inline-size: 100%;
+}
+:global(.auth .content:after) {
+  content: "";
+  display: block;
+  block-size: 30px;
+}
+:global(.auth .footer) {
+  background-color: #fff;
+  min-inline-size: 300px;
+  position: absolute;
+  font-size: 12px;
+  inset-inline: 0;
+  inset-block-end: 0;
+  block-size: 34px;
+  overflow: hidden;
+}
+:global(.auth .footer .footer-container) {
+  padding-block: 9px;
+  padding-inline: 80px;
+  margin-block: 0;
+  margin-inline: auto;
+  box-sizing: border-box;
+}
+@media only screen and (max-width: 560px) {
+  :global(.auth .footer .footer-container) {
+    inline-size: 400px;
+    padding-inline: 20px;
+  }
+}
+@media only screen and (max-width: 400px) {
+  :global(.auth .footer .footer-container) {
+    inline-size: 100%;
+  }
+}
+:global(.auth .footer a) {
+  color: #6e6e78;
+}
+:global(.auth .footer a:active,
+.auth .footer a:link,
+.auth .footer a:visited) {
+  text-decoration: none;
+  color: #6e6e78;
+}
+:global(.auth .footer a:focus,
+.auth .footer a:hover) {
+  text-decoration: underline;
+  color: #6e6e78;
+}
+:global(.auth .footer .copyright) {
+  float: inline-start;
+  color: var(--ColorTextSub);
+}
+:global(.auth .footer .copyright a) {
+  color: var(--ColorTextSub);
+}
```

Comment:
`color: var(--ColorTextSub);` Corrects the a11y issue in the footer (confirmed this passes axe test.